app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1685,6 +1671,15 @@ BrowserWorld::TickWorld() {
   };
 }
 
+HandMeshStrategy
+SelectHandMeshRenderingStrategy() {
+#if defined(PICOXR) || defined(SPACES)
+  return HandMeshStrategy::Spheres;
+#else
+  return HandMeshStrategy::Skinned;
+#endif
+}
+
```

Comment:
Ok. Initially, I thought that we would use the `strategy` for something else, but at the end it was only used for instancing the object. So I agree it doesn't make much sense now.

Reply:
> Instead of ifdefs we should in general check capabilities. So I'd do something like
> 
> ```
> if (FB_extension supported)
>   return X
> else if (MSFT_extension supported)
>   return Y
> ...
> ```

The problem is that neither of these two strategies depend on any extension. This will only be true for the MSFT one.
We have a generic, default one: Skinned. But then we have a special case for PICOXR and SPACES that use the Spheres one because their hand tracking data is bad/noisy.

So I don't see how we are going to remove the ifdefs for these cases :/